dist/comment_style.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import argparse, os, re, subprocess, sys
+
+def print_msg(file_name, line_num, line, multiline):
+    print("Illegal " + multiline + "comment in " + file_name + ":" + str(line_num - 1) + " " +
```

Comment:
I'd recommend an f-string here:
`f"Illegal {multiline} comment in {file_name}:{line_num - 1} {line.strip()}"`
`.strip` also trims all whitespace by default so I don't think we need the `\n` arg here

Reply:
I don't think we can do that, since f-strings are a Python 3.6 addition and I think at least one of the builders is 3.4.

Reply:
No worries. Our [min python version](https://wiki.corp.mongodb.com/display/WT/Tool%2C+compiler+and+language+standard+versions) is (in theory) 3.7 but happy to drop it if we know there are issues.

Reply:
Yeah I did it for another PR last year and broke a pile of stuff unfortunately.